fictadvisor-api/src/v2/utils/documentation/telegramGroup/getTelegramGroups.ts

```diff
@@ -0,0 +1,21 @@
+import { TelegramGroupResponse } from '@fictadvisor/utils/responses';
+import { ApiDocumentationParams } from 'src/v2/utils/documentation/decorators';
+
+export const TelegramGroupDocumentationGetTgGroups: ApiDocumentationParams = {
+  ok: {
+    type: TelegramGroupResponse,
+  },
```

Comment:
actual response contain an array of objects of this reponse type
create a class `TelegramGroupsResponse` in `responses` directory and write this:
```
{
  telagramGroups: [TelegramGroupResponse],
}
```
add also `@ApiProperty()` decorator with a description